DocumentedExamples/NearestNeighbourDistance.ipynb

```diff
@@ -0,0 +1,1730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.                                 start_time = '1978-01', end_time = '1978-03',
</pre><p> </p><p> </p><p>what is <code>var_ice</code>? should we give it a more verbose name?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
The text block immediately above examples that this refers to monthly sea ice concentration data from the 4th cycle of ACCESS-OM2-01. This is also explained at the beginning of the notebook. `var_ice` refers to variable ice because this variable contains sea ice data. Are we meant to do examples assuming people only read the code chunks rather than the full notebook?